apps/etterlatte-behandling/src/main/kotlin/behandling/jobs/SaksbehandlerJobService.kt

```diff
@@ -8,19 +8,30 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.coroutineScope
 import kotlinx.coroutines.newSingleThreadContext
 import kotlinx.coroutines.runBlocking
+import no.nav.etterlatte.behandling.klienter.NavAnsattKlient
 import no.nav.etterlatte.behandling.klienter.SaksbehandlerInfo
-import no.nav.etterlatte.config.ApplicationContext
+import no.nav.etterlatte.saksbehandler.SaksbehandlerInfoDao
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
-import java.time.Duration
 import kotlin.time.measureTime
 
-@OptIn(DelicateCoroutinesApi::class)
-internal fun populerSaksbehandlereMedNavn(context: ApplicationContext) {
-    val logger = LoggerFactory.getLogger("Saksbehandlerdatanavn")
-    Thread.sleep(Duration.ofMinutes(3L).toMillis())
-    if (context.leaderElectionKlient.isLeader()) {
-        logger.info("Henting av Saksbehandlerdatanavn jobb leader is true, starter jobb")
+val ugyldigeIdenter =
+    listOf(
+        "PESYS",
+        "EY",
+        "GJENOPPRETTA",
+    )
+
+val SAKSBEHANDLERPATTERN = Regex("[a-zA-Z]\\d{6}")
+
+class SaksbehandlerJobService(
+    private val saksbehandlerInfoDao: SaksbehandlerInfoDao,
+    private val navAnsattKlient: NavAnsattKlient,
+) {
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    @OptIn(DelicateCoroutinesApi::class)
+    fun run() {
         val subCoroutineExceptionHandler =
             CoroutineExceptionHandler { _, exception ->
                 logger.error("Saksbehandlerdatanavn feilet se exception $exception")
```

Comment:
```suggestion
                logger.error("Saksbehandlerdatanavn feilet se exception", exception)
```
så får vi med stacktrace etc.

Reply:
Lurt! https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3892/commits/99deacd1ce5fbeeb94a62c213c10d82a19bf3365